modules/multi-arch-deleting-podplacmement-config-using-cli.adoc

```diff
@@ -0,0 +1,58 @@
+//Module included in the following assemblies
```

Comment:
^ be careful with the name of some files, `s/podplacmement/podplacement`

Reply:
This is a great catch! 
Updated the filenames. 
Thank you :)